corehq/apps/app_manager/static/app_manager/js/releases/releases.js

```diff
@@ -376,6 +378,13 @@ hqDefine('app_manager/js/releases/releases', function () {
             });
         };
 
+        self.releaseModeVisible = ko.observable(true);
+        self.releaseLocked = ko.observable(true);
```

Comment:
what is the difference between releasesEnabled and releaseLocked. By looking at their usages I really can't tell their purpose.

Reply:
We are using `releasesEnabled` only now, removed the references for `releaseLocked`